ReactNative/ReactTwitter/core/views/tweetsListItem.js

```diff
@@ -24,16 +24,18 @@ var TweetsListItem = React.createClass({
   },
 
   render () {
-    var formattedTime = novodaDataFormat.longTime(this.props.time)
+    var formattedTime = novodaDataFormat.elapsedTime(this.props.time)
     return (
       <TouchableHighlight onPress={() => this._tweetSelected(this.props.id)}
         underlayColor='#dddddd'>
          <View>
           <View style={styles.rowContainer}>
-            <Image style={styles.tweet_avatar} source={{ uri: this.props.author_avatar }} />
+            <AvatarView uri={this.props.author_avatar} size={60} />
```

Comment:
AvatarView is a custom component that I have created. I decided to pass the size as a props so that internally it uses it for both the width and the height.